### DISCUSSION TITLE
Naim NDX

### DISCUSSION THREAD

**Discussion by jeandelcapitaine** [eizedev/AirConnect-Synology#91] [Q&A] [answered]
Hello,

First of all, thank you very much for this great package !

I added Naim 1st NDX streamer in the list of supported models. Model Number is 20-015-0011. So now, Audio Station can stream radios to this device and then to my McIntosh Amp.

The only drawback is that the native Focal/Naim Ipad App is not able to stream radios anymore to the NDX. The App tries to do some preloading stuff and then just stops. I suspect some conflict between Airconnect and the App.

Any idea on how to debug that ?

Thanks again.

**Comment by eizedev** [comment]
Hi, thanks :) 
I am not at home during the holidays but, perhaps, the discussion in these issues will help you?

https://github.com/philippe44/AirConnect/issues/356

and

https://github.com/philippe44/AirConnect/issues/19#issuecomment-352031924

With f.e. https://debookee.com/ you can trace upnp traffic and if you set logging for airupnp to debug (see Readme) you could also get more detailed output on your Synology NAS device.

I think i cannot help you with the naim & focal ipad app and the naim devices, but you could also open an issue at the [AirConnect](https://github.com/philippe44/AirConnect) repository and add an debug log of airupnp and/or a network trace from a tracing tool.

Merry Christmas / Happy Holidays

René

**Comment by jeandelcapitaine** [comment]
Hello, nope. That would be a different topic. Not a quality issue, more a
conflict between the native Naïm app and AirConnect I reckon.

Thanks

JDEC

**Comment by eizedev** [comment]
If i should add your model number '20-015-0011' to the pre generated list of supported devices, please let me know.

For your initial problem, you could open an issue at the [AirConnect](https://github.com/philippe44/AirConnect) repository and add an debug log of airupnp to it. The developer and owner usually responds really quickly (much faster than me ;-))

**Comment by eizedev** [comment, marked as answer]
Closing this out since there wasn't a follow-up - no worries either way. If you (or anyone else with a Naim NDX) want the model number `20-015-0011` picked up without waiting on a default-filter change, you can add it yourself in `airconnect.conf`: append it to the existing `FILTER_AIRPLAY2_DEVICES` list, e.g. `FILTER_AIRPLAY2_DEVICES="<NULL>,S1,S3,S5,S9,S12,ZP80,ZP90,S15,ZP100,ZP120,1.0,LibreWireless,Fitzwilliam,2.2.6,AllShare1.0,20-015-0011"`, then restart the package. See [doc/CONFIG.md](https://github.com/eizedev/AirConnect-Synology/blob/master/doc/CONFIG.md#supported-upnp-speakers) for the full config reference.
